geode-core/src/main/java/org/apache/geode/internal/cache/wan/parallel/ParallelGatewaySenderQueue.java

```diff
@@ -790,17 +790,17 @@ public boolean put(Object object) throws InterruptedException, CacheException {
           LocalRegion.setThreadInitLevelRequirement(oldLevel);
         }
       } else {
-        boolean thisbucketDestroyed = false;
+        boolean thisBucketDestroyed;
```

Comment:
just a thought, would this simplify to :
boolean thisBucketDestroyed = brq.isDestroyed();
if (!isDREvent) {
  thisBucketDestroyed |= prQ.getColocatedWithRegion()...;
}

Reply:
Changed!.